CareKitStore/CareKitStore/Protocols/Tasks/OCKAnyTaskStore.swift

```diff
@@ -133,7 +133,7 @@ public protocol OCKAnyTaskStore: OCKAnyReadOnlyTaskStore {
 
 public extension OCKAnyReadOnlyTaskStore {
     func fetchAnyTask(withID id: String, callbackQueue: DispatchQueue = .main, completion: @escaping OCKResultClosure<OCKAnyTask>) {
-        var query = OCKTaskQuery(id: id)
+        var query = OCKTaskQuery(for: Date())
```

Comment:
Switched this so it's alignment with the other Entity fetches. It was currently setting the `id` twice instead of querying the current version for today. Without the change, it's possible for this fetch to return a future version of the task since it would have a future `effectiveDate`.